### PR TITLE
PR Add opencollective badges and sponsors/backers sections.

### PR DESCRIPTION
**Pull request by rlaverde**
See: https://github.com/spyder-ide/spyder/wiki/Anaconda-stopped-funding-Spyder

### CODE REVIEW

**Comment by rlaverde** [comment]
@goanpeca @ccordoba12 Should I also add the text in https://github.com/spyder-ide/spyder/pull/5804?

**Comment by andfoy** [comment]
@rlaverde It seems that all repositories have also the Anaconda disclaimer

**Comment by goanpeca** [comment]
Yes @rlaverde I added the disclaimer on all repos

File: README.rst [hunk 1]
```diff
@@ -62,6 +62,22 @@ Overview
 .. image:: https://github.com/spyder-ide/spyder-reports/blob/master/doc/reports_screenshot.png
    :alt: Reports Screenshot
 
+# Important Announcement: Spyder is unfunded!
+---
```
Comment (goanpeca) [inline README.rst]: Full line, please... aesthetics

**Pull request merged**